calico-cloud/image-assurance/understanding-scan-results.mdx

```diff
@@ -0,0 +1,125 @@
+---
+description: Identify vulnerabilities in workloads to Kubernetes clusters.
```

Comment:
You'll want to change these to be appropriate for each page.

Reply:
fixed with https://github.com/tigera/docs/pull/676/commits/16d2c8cab26cc48b8ee6446e1aa77b017e8294ac